runtime/src/asset.rs

```diff
@@ -582,14 +589,29 @@ decl_module! {
                 // New total supply must be valid
                 token.total_supply = updated_total_supply;
             }
-
-            // After checks are ensured introduce side effects
+            let round = Self::funding_round(&ticker);
+            // Update the total token balance issued in this funding round.
+            let issued_in_this_round = Self::issued_in_funding_round((ticker.clone(), round.clone()));
```

Comment:
Pls check `<IssueInFundingRound<T>>::mutate`, it will allow you to reduce the number of `clone()` in the key.

Reply:
btw, you can also do `let tupple = (ticker.clone(), round.clone());` and then use `&tupple` where you want to pass the tupple. It can save a few clones. As Miguel mentioned, using `mutate` is better in this case. You can combine mutates with tupple references for added benefits some times :)

Reply:
Since it's not possible to return a `Result` from the closure in `mutate`, and since we use the mutated value further down in the same function, may I suggest an intermediate approach with getting and inserting while using a reference to the pair containing the cloned `ticker` and `round`?